man/bundle-config.ronn

```diff
@@ -155,6 +155,14 @@ learn more about their operation in [bundle install(1)][bundle-install].
    The number of redirects allowed for network requests. Defaults to `5`.
 * `timeout` (`BUNDLE_TIMEOUT`):
    The seconds allowed before timing out for network requests. Defaults to `10`.
+* `force_ruby_platform` (`BUNDLE_FORCE_RUBY_PLATFORM`):
+   Force bundler to install the `ruby` platform version of gems, instead of
+   platform-specific gems.
```

Comment:
I feel like it's valuable to let the user know what the results of this setting will be. How do you feel about saying something like "Ignore the current machine's platform and install only `ruby` platform gems. As a result, gems with native extensions will be compiled from source."